cedar-policy/src/ffi/is_authorized.rs

```diff
@@ -446,7 +446,7 @@ pub struct AuthorizationCall {
     principal: Option<JsonValueWithNoDuplicateKeys>,
     /// The action the principal is taking
     #[cfg_attr(feature = "wasm", tsify(type = "{type: string, id: string}"))]
-    action: JsonValueWithNoDuplicateKeys,
+    action: Option<JsonValueWithNoDuplicateKeys>,
     /// The resource being acted on by the principal
     #[cfg_attr(feature = "wasm", tsify(type = "{type: string, id: string}"))]
     resource: Option<JsonValueWithNoDuplicateKeys>,
```

Comment:
I think the optional fields here will be confusing for people who aren't using partial auth. I'd prefer adding a new `PartialAuthorizationCall` type with this signature.